eng/Version.Details.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Dependencies>
   <ProductDependencies>
+    <Dependency Name="Microsoft.Bcl.AsyncInterfaces" Version="9.0.0-preview.2.24111.9">
```

Comment:
There are entries for dotnet-format in Version.Details.xml that are no longer needed.

Reply:
Let me test if source build would happy